tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers_test.ts

```diff
@@ -507,4 +512,65 @@ describe('Debugger reducers', () => {
     const nextState = reducers(state, actions.executionScrollRight());
     expect(nextState.executions.scrollBeginIndex).toEqual(1450);
   });
+
+  it(`Updates states on executionDigestFocused`, () => {
```

Comment:
Could you add a spec that tests `scrollBeginIndex` + `displayIndex` where `scrollBeginIndex > 0`?

Reply:
Good suggestion. Done.